rust/agents/relayer/src/msg/gas_payment/mod.rs

```diff
@@ -195,7 +194,7 @@ mod test {
         test_utils::run_test_db(|db| async move {
             let hyperlane_db =
                 HyperlaneRocksDB::new(&HyperlaneDomain::new_test_domain("test_no_match"), db);
-            let matching_list = serde_json::from_str(r#"[{"originDomain": 234}]"#).unwrap();
+            let matching_list = serde_json::from_str(r#"[{"origindomain": 234}]"#).unwrap();
```

Comment:
is this necessary? assuming this is from Mattie's config change but curious if this implies that `originDomain` doesn't work anymore

Reply:
I think what tests is whether the cli is robust against casing inconsistencies and that `originDomain` will still work